lib/tarantool_server.py

```diff
@@ -239,8 +239,6 @@ def flush(self, ts, command_log, command_exe):
 
     def exec_loop(self, ts):
         self.write_result_file_version_line()
-        if not self.execute_pretest_clean(ts):
```

Comment:
Remove the unused method `execute_pretest_clean()` as well.

Reply:
Right, removed unused method definition.